deploy/test/mock_manager.ts

```diff
@@ -0,0 +1,17 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
```

Comment:
mock_manager is deploying smart contract MockManager.sol rather than Manager.sol which allows to skip multiple checks in Manager.sol and really check just what is necessary for the test